src/internal/messageBridge.ts

```diff
@@ -27,11 +28,19 @@ export function postRuneEvent(data: RuneGameEvent) {
   }
 
   // Game is not running in iframe, don't try to send a message and notify user
-  if (globalThis.parent === (globalThis as typeof window)) {
+  if (
+    globalThis.parent === (globalThis as typeof window) &&
+    data.type !== "WINDOW_ERR"
+  ) {
     // eslint-disable-next-line no-console
     console.error(
       "Games using the Rune SDK must be tested using Rune CLI. See https://github.com/rune/rune-games-cli for instructions."
     )
+
+    showFullScreenError(
+      'Games using the Rune SDK must be tested using Rune CLI. See <a href="https://github.com/rune/rune-games-cli">https://github.com/rune/rune-games-cli</a> for instructions.'
```

Comment:
Might be nice to automatically convert the links? That’ll make our life easier as we might otherwise forget to do it.

Reply:
Perhaps we could also make showFullScreenError throw and thereby get rid of the throwing/logging of an almost identical string. Then we don’t have to keep them in sync :)